docs/release_process.md

```diff
@@ -0,0 +1,198 @@
+## Release Cadence
+
+Community EE is to be built and published on the next day and/or the day after ansible-core is released.
+
+## EE tag versioning
+
+Versioning would be the core tag + a patch number, e.g
+
+    - EE with core `2.16.2` comes out -> `community-ee:2.16.2-1`
+    - EE with core `2.16.3` comes out -> `community-ee:2.16.3-1`
+
+
+## Dependencies
+
+-   Podman/Docker
+
+##  Credentials
+
+- Access to  https://github.com/ansible-community/images repo
+- Access to ghcr.io
+
+
+## Build steps for Community-ee-base
+
+1. Go to the working directory.
+
+`cd images/execution-environments/community-ee-base`
+
+2. Create a new release branch
+
+`git checkout -b base-<ansible-core-version-base-ee-version>`
+
+3. Edit the `execution-environment.yml` file to the right `collection version` and `ansible-core version`. Get the correct [ansible-core](https://pypi.org/project/ansible-core/) version and [ansible-<version>.deps](https://github.com/ansible-community/ansible-build-data/blob/main/9/ansible-9.0.1.deps) and edit accordinly.
+
+`vim execution-environments/community-ee-base/execution-environment.yml`
+
+```
+version: <base-community-ee with the specific ansible core>
+images:
+  base_image:
+    name: quay.io/fedora/fedora:latest
+dependencies:
+  ansible_core:
+    package_pip: ansible-core==<ansible-core-version>
```

Comment:
There's still my comment from last time:

> One thing that I find strange is that the `release_process.md` document tells the RM to **replace** the EE definition with the content described in this file, instead of simply updating the relevant numbers. Shouldn't the main source of truth be the EE definitions included in this repository?